functional/FunctionalTest.java

```diff
@@ -2473,12 +2478,23 @@ public static void main(String[] args) {
       client = new MinioClient(endpoint, accessKey, secretKey);
       // Enable trace for debugging.
       // client.traceOn(System.out);
-      FunctionalTest.runTests();
-
-      // Run fast test with region parameter passed to the constructor
-      client = new MinioClient(endpoint, accessKey, secretKey, region);
-      FunctionalTest.runFastTests();
 
+      // For mint environment, run tests based on mint mode
+      if (mintEnv) {
+        if (mintMode != null && mintMode.equals("full")) {
+          FunctionalTest.runTests();
+        } else {
+          FunctionalTest.runQuickTests();
+        }
+      } else {
+        FunctionalTest.runTests();
+
+        // Get new bucket name to avoid minio azure gateway failure.
+        bucketName = getRandomName();
```

Comment:
Looks like this line doesn't have an effect.

Reply:
Unable to understand